docs/datasets/index.md

```diff
@@ -1,21 +1,39 @@
 # Datasets
-SciCat datasets are sets of meta data and can include several files which e.g. comprise a self-contained measurement - which is fully customizeable during ingestion of meta data. Users can search, view, list the meta data of a dataset. 
+SciCat datasets are sets of metadata and can include several files which e.g. comprise a self-contained measurement - which is fully customizeable during ingestion of metadata. Users can search and view different formats (e.g. in tree, tables or as JSON) of the dataset and list them. 
 
-To group and tag datasets is depicted [here](grouping_tagging_ds.md). Datasets can also be issued to be **published**: either removing the restricted view or triggering the process of obtaining a DOI for the selected datasets, see description of [publication of SciCat datasets](Publishing.md).
+## Features
+* A very handy feature is to group and tag datasets. Find [here](grouping_tagging_ds.md) more details how to do it. 
+* Datasets can also be selected for several actions. One such action is the *publication* of that selection. For more details see [publication of SciCat datasets](Publishing.md). Generally, actions depend on what is implemented at your site and can cover a wide range from comprising them into a new datacollection of a custom type [(see advanced documentation)](../datasets/datasetTypes.md) to using that selection of datasets to run an analysis on them.
 
-### How to query datasets
+## How to search for datasets
+You can use several places to query your datasets: 
 
+* from the top the search bar and
```

Comment:
suggestions 
the search bar at the top of the page provides a quick free text search